odoo/openupgrade/doc/source/modules110-120.rst

```diff
@@ -27,7 +27,7 @@ missing in the new release are marked with |del|.
 +--------------------------------------------+-------------------------------------------------+
 |account_analytic_default                    |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-| |del| account_asset                        |                                                 |
+| |del| account_asset                        | Replaced by OCA [#account_asset]_               |
```

Comment:
s/OCA/OCA module, and even better to put directly `account_asset_management` in the link name

Reply:
the link name it's not visible, so I prefer the shorter one (the longer one doesn't fit)